step-release-vis/src/app/services/environment_test.ts

```diff
@@ -210,6 +210,108 @@ describe('EnvironmentService', () => {
     expect(result.snapshot).toEqual([]);
   });
 
+  it('#addSnapshotToPolygons snapshot with the 2 same existing candidates', () => {
```

Comment:
maybe add more describe blocks to structure the testes a bit more? (one for all of these would be nice)

Reply:
do you mean to have a 'describe' for each set of tests for a function?

Reply:
oh, sorry. I already added for all and now I have just seen the comment after pushing :(

Reply:
I'll change ASAP

Reply:
no need ;)